### PR TITLE
moved projteam and twitter

### PR DESCRIPTION
**Pull request by mellolikejello**

### CODE REVIEW

File: people/2015/spring/mellolikejello.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ hw:
   commarchpreso: https://mellolikejello.wordpress.com/2015/03/11/commarch-presentation/
   curriculum: https://mellolikejello.wordpress.com/2015/03/18/curriculum/
   teamprop2: https://mellolikejello.wordpress.com/2015/03/20/anglegators-team-proposal/
+  projteam: amm4108, RedTwo
```
Comment (Nolski) [inline people/2015/spring/mellolikejello.yaml]: I think that this should be a link to the blog post which contains your team. @decause @liam-middlebrook am I correct here?

Comment (liam-middlebrook) [inline people/2015/spring/mellolikejello.yaml]: I think that makes more sense as a blog post. 

But in the case it is supposed to be a list I think you should do an actual list, maybe something like:

```
  projteam: ['amm4108', 'RedTwo']
```

**Comment by mellolikejello** [comment]
@Nolski the blog post would just be teamprop2

I'm pretty sure a list was expected, but I'm not sure what it will look like on the participants page (since they're usually clickable links)

**Comment by Nolski** [comment]
@mellolikejello Ahhh I see now. Yeah that makes sense. I would take @liam-middlebrook's advice and format it like a python list.

**Comment by decause** [comment]
:jeans: :panda_face:

**Pull request merged**